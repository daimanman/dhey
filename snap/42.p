{"C":1,"Z":"","N":1,"Remark":"压力测试","Url":"http://localhost:9188/go1","Method":"POST","TaskId":42,"Type":"FIXED","FileId":"4","Status":0,"Err":"","QH":{},"P":{"info":"TEsss颠三倒四ST","name":"DXwwM"},"Payload":"","StartTime":"2018-08-20T11:05:38.614335+08:00","Duration":0}