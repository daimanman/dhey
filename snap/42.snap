{"AvgTotal":0.3459206,"Fastest":0.3459206,"Slowest":0.3459206,"Average":0.3459206,"Rps":2.890836798964849,"AvgConn":0.0040037,"AvgDNS":0.0029998,"AvgReq":0,"AvgRes":0,"AvgDelay":0.3409071,"ConnMax":0.0029998,"ConnMin":0.0029998,"DnsMax":0.0029998,"DnsMin":0.0029998,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.3409071,"DelayMin":0.3409071,"Lats":[0.3459206],"ConnLats":[0.0029998],"DnsLats":[0.0029998],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.3409071],"Total":345920600,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":45,"SizeReq":45,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.3459206,"Count":1,"Frequentcy":1},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0},{"Mark":0.3459206,"Count":0,"Frequentcy":0}],"TaskId":42,"Duration":0.3459206}